bodhi/server/models.py

```diff
@@ -2553,7 +2553,6 @@ def set_request(self, db, action, username):
         # keep the status at testing
         elif self.request in (UpdateRequest.stable, UpdateRequest.batched) and \
                 self.status is UpdateStatus.testing and action is UpdateRequest.revoke:
-            self.status = UpdateStatus.testing
```

Comment:
hahah that's pretty funny. Nice find.

Reply:
At first I assumed some magic here, but turns out just a redundant assignment :)